pkg/client/cache/listwatch_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cache
+package cache_test
```

Comment:
@sttts I suspect we're going to end up with a `pkg/client/testing` like we did for `apimachinery`.